<launch>
<group ns="developer">

	<param name="tf_prefix" value="developer" />
	
	<rosparam command="load" ns="move_group" file="$(dirname)/../config/controllers_D.yaml" />
	<!-- Launch moveit_planning_execution.launch from motoman_gp4_moveit_config package under the 'architect' namespace -->
    	<include file="$(find motoman_gp4_moveit_config)/launch/moveit_planning_execution.launch">
		<arg name="robot_ip" value="192.168.1.74" />
		<arg name="sim" value="false" />
		<arg name="default_action_ns" value="false" />
    	</include>

	<!-- Start move_for_lego node -->
    	<node name="move_for_lego_node" pkg="lego_moveit" type="move_for_lego" output="screen">
		<param name="group_name" type="str" value="manipulator" />
	</node>

	<node name="enable_robot" pkg="lego_moveit" type="rosservice_enable.py" output="screen">
		<param name="namespace" type="str" value="developer" />
	</node>
</group>
</launch>
